web/massastation/src/main.tsx

```diff
@@ -17,9 +17,11 @@ import { Error, NotFound, Index, Base, Store } from './pages/index';
 import mockServer from './mirage/server.js';
 
 const baseURL = import.meta.env.VITE_BASE_APP;
+const baseENV = import.meta.env.VITE_ENV;
 
-if ([ENV.DEV, ENV.TEST].includes(baseURL)) {
-  mockServer(baseURL);
+// Add ENV.STANDALONE to the array to enable MirageJS
```

Comment:
there is no standalone mode for MassaStation, this comment was specific to the wallet plugin


```suggestion
```

Reply:
@maxime-carabina can answer :)

Reply:
VITE_ENV: test or dev or prod
VITE_BASE_APP: base path URL of the application

Reply:
Thanks @Thykof